### DISCUSSION TITLE
How do I fix this error in linux on a clean install.

### DISCUSSION THREAD

**Discussion by superskirv** [KoboldAI/KoboldAI-Client#434] [Q&A]
Getting weird error on linux mint. Not Sure why this is happening on running a clean install with ./play.sh. The drive is ntfs formated and I saw some one close an issue similar but I dont know what the solution was. I can not reformat the drive, if that is the issue, as I use it with dual booting windows and linux.

```
Downloading  (5) ━━━━━━╸━━━━━━━━━━━━━╸━━ 163.2MB /   2.6GB @  90.7MB/s libgfortran4               2.6s
Extracting   (4) ━━━━━━━━━━╸━━╸━━━━━━━━━      74 /     102             gettext                    2.5s
critical libmamba Can't create 'man/man3/Parse::CPAN::Meta.3'
perl-5.32.1-7_hd590300_perl5.conda extraction failed
```
and later in the installer...
```
pytorch                                              1.4GB @  42.2MB/s 31.4s
critical libmamba Found incorrect download: perl. Aborting
```
then it ends with this last bit...
```
Transaction starting
perl                                                13.3MB @  17.2MB/s  0.8s
[+] 0.8s
Downloading      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  13.3MB /  13.3MB                            0.8s
Extracting   (1) ━╸━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━       0 /       1 perl                       0.0s
critical libmamba Can't create 'man/man3/Parse::CPAN::Meta.3'
perl-5.32.1-7_hd590300_perl5.conda extraction failed
critical libmamba Found incorrect download: perl. Aborting
/tmp/mambafL3EOAHelzw: line 2: /media/username/Ai: No such file or directory
/tmp/mambafL3EOAHelzw: line 3: micromamba: command not found
/tmp/mambafL3EOAHelzw: line 5: exec: python: not found

```

**Comment by superskirv** [comment]
Wait, I might have found the issue. Drive is called "Ai Drive" so the space might be screwing things up. Deleted everything and trying again with out the space in the drive name. I just `ls -S "real location" "fake_location"` the drive location so it should work, crossing fingers.

Edit: Nope didnt change anything.

**Comment by henk717** [comment]
Mounted filesystems are not supported by the runtime if their filesystem is not compatible with Linux filenames.

**Reply by superskirv** [reply]
I just downloaded a different distro and it works just fine. Its clearly something with the path but I cant tell what. Every other ai thing I have works just fine from that drive in linux. But its all good, thank anyway.
